ui/lib/kv/addon/components/kv-page-header.hbs

```diff
@@ -29,15 +27,11 @@
 
 {{#if (or (has-block "toolbarFilters") (has-block "toolbarActions"))}}
   <Toolbar>
-    {{#if (has-block "toolbarFilters")}}
-      <ToolbarFilters>
-        {{yield to="toolbarFilters"}}
-      </ToolbarFilters>
-    {{/if}}
+    <ToolbarFilters>
```

Comment:
I kind of remember an issue of styling showing the ToolbarFilters html (basically just a nav and class) for views that showed no filters. Just making sure we double checked there were no styling issues if toolbarFilters was empty.